lib/rubocop/cop/naming/rescued_exceptions_variable_name.rb

```diff
@@ -150,6 +153,13 @@ def message(node)
           preferred_name = preferred_name(offending_name)
           format(MSG, preferred: preferred_name, bad: offending_name)
         end
+
+        def shadowed_variable_name?(node)
+          node.each_descendant(:lvar) do |n|
+            return true if n.children.first.to_s == preferred_name(n)
+          end
+          false
```

Comment:
It seems that it can be simplified by using `any?`.

```diff
         def shadowed_variable_name?(node)
-          node.each_descendant(:lvar) do |n|
-            return true if n.children.first.to_s == preferred_name(n)
-          end
-          false
+          node.each_descendant(:lvar).any? { |n| n.children.first.to_s == preferred_name(n) }
         end
```